breathecode/authenticate/tests/urls/tests_academy_member_id.py

```diff
@@ -542,39 +626,189 @@ def test_academy_member_id__put__passing_email(self):
+            self.bc.request.set_headers(academy=n)
+            model = self.bc.database.create(authenticate=True,
+                                            capability='crud_member',
+                                            role='role',
+                                            profile_academy=profile_academy)
+
+            url = reverse_lazy('authenticate:academy_member_id', kwargs={'user_id_or_email': n})
+            response = self.client.put(url)
+
+            json = response.json()
+            expected = {'detail': 'email-not-found', 'status_code': 404}
+
+            self.assertEqual(json, expected)
+            self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    @patch('os.getenv', MagicMock(return_value='https://dotdotdotdotdot.dot'))
+    def test_academy_member_id__put__without_first_name(self):
+        """Test /academy/:id/member/:id"""
+        profile_academy = {
+            'first_name': self.bc.fake.first_name(),
+            'last_name': self.bc.fake.last_name(),
+            'email': self.bc.fake.email()
+        }
+        role = 'konan'
+        self.bc.request.set_headers(academy=1)
+        model = self.bc.database.create(authenticate=True,
+                                        role=role,
+                                        capability='crud_member',
+                                        profile_academy=profile_academy)
+
+        url = reverse_lazy('authenticate:academy_member_id', kwargs={'user_id_or_email': '2'})
+
+        data = {'role': role, 'last_name': self.bc.fake.last_name(), 'email': self.bc.fake.email()}
+        response = self.client.put(url, data, format='json')
+
+        json = response.json()
+        expected = {'detail': 'first-name-not-found', 'status_code': 400}
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    @patch('os.getenv', MagicMock(return_value='https://dotdotdotdotdot.dot'))
+    def test_academy_member_id__put__without_last_name(self):
+        """Test /academy/:id/member/:id"""
+        profile_academy = {
+            'first_name': self.bc.fake.first_name(),
+            'last_name': self.bc.fake.last_name(),
+            'email': self.bc.fake.email()
+        }
+        role = 'konan'
+        self.bc.request.set_headers(academy=1)
+        model = self.bc.database.create(authenticate=True,
+                                        role=role,
+                                        capability='crud_member',
+                                        profile_academy=profile_academy)
+
+        url = reverse_lazy('authenticate:academy_member_id', kwargs={'user_id_or_email': '2'})
+
+        data = {'role': role, 'first_name': self.bc.fake.first_name(), 'email': self.bc.fake.email()}
+        response = self.client.put(url, data, format='json')
+
+        json = response.json()
+        expected = {'detail': 'last-name-not-found', 'status_code': 400}
```

Comment:
This appear 3 times, you cover one of 3 apparitions of this exception, it is in a view and 2 serializers and it cover just one of 3 apparition